integr8ly/cloud-resource-operator
# cut release 0.21.0

## Overview

Jira: https://issues.redhat.com/browse/INTLY-7234

## Verification

- Run `make cluster/prepare`
- Run `make run`
- Ensure `make cluster/seed/managed/redis` & `make cluster/seed/managed/postgres` & `make cluster/seed/managed/blobstorage` deploy to aws.
